fix exception handling on python 2
where socket.error is not a subclass of OSError

--- src/hpp/corbaserver/client.py	2022-05-25 20:57:17.000000000 +0200
+++ /home/gsaurel/local/humanoid-path-planner/hpp-corbaserver/src/hpp/corbaserver/client.py	2022-07-27 19:08:55.338566661 +0200
@@ -163,16 +163,14 @@
     import os
 
     try:
+        import socket
         import rospy
 
-        try:
-            # Check is ROS master is reachable.
-            if rospy.client.get_master().target is not None:
-                _host = rospy.get_param("/hpp/host", _host)
-                _port = rospy.get_param("/hpp/port", _port)
-        except OSError:
-            pass
-    except ImportError:
+        # Check is ROS master is reachable.
+        if rospy.client.get_master().target is not None:
+            _host = rospy.get_param("/hpp/host", _host)
+            _port = rospy.get_param("/hpp/port", _port)
+    except (ImportError, OSError, socket.error):
         pass
     _host = os.getenv("HPP_HOST", _host)
     _port = os.getenv("HPP_PORT", _port)
